WooCommerce/Classes/ViewRelated/Products/Edit Product/Shipping Settings/Product+ShippingSettingsViewModels.swift

```diff
@@ -22,7 +23,8 @@ extension Product {
         let unit = shippingSettingsService.dimensionUnit ?? ""
         return UnitInputViewModel(title: title,
                                   unit: unit,
-                                  value: length.isEmpty ? "0": length,
+                                  value: length.isEmpty ? "": length,
```

Comment:
nit: I think this could be `value: length` since `length.isEmpty == true` means it's `""`?